NEWS.md

```diff
@@ -26,12 +26,23 @@
   additional column to be added in the last position in the resulting data frame
   that will identify the source data frame.
   ([#2649](https://github.com/JuliaData/DataFrames.jl/pull/2649))
+* since Julia 1.7 using broadcasting assignment on a `DataFrames` column
+  selected as a property is allowed when it does not exist in a data frame
+  and it allocates a fresh column
```

Comment:
```suggestion
* since Julia 1.7 using broadcasted assignment on a `DataFrame` column
  selected as a property (e.g. `df.col .= 1`) is allowed when column does not
  exist and it allocates a fresh column
```